README.md

```diff
@@ -1,6 +1,13 @@
 # raspberry-pi
 UBports Raspberry Pi Focus Group. Join the discussion on [our forum](https://forums.ubports.com/category/43/ut-for-raspberry-pi) and our [Telegram focus group](https://t.me/UBports_pi).
 
+## Contents
```

Comment:
Is a table of contents really necessary in such a small file?

Reply:
Two reasons why i've added the toc:
1. hopefully it will not stay small for long time ;-) - but yes then we should split it up either ways
2. i wanted the people to see there's a faq at a glance, but not putting it in front of the other sections...